packages/blockchain/ada/src/lib/services/ada.wallet.ts

```diff
@@ -0,0 +1,44 @@
+import { generateAddress, generatePrivateKey, generateXPublicKey } from './ada.crypto'
```

Comment:
Why are these implementations in separate file?

Reply:
cc @jakcinmarina Any progress there?

Reply:
yeah I moved it ada.wallet file